_data/simple-icons.json

```
@@ -3390,6 +3390,11 @@
             "hex": "000000",
             "source": "https://www.crew-united.com"
         },
+        {
+            "title": "Cricbuzz",
+            "hex": "009270",
+            "source": "https://www.cricbuzz.com/images/apple-touch-icon.png"
```

Also, you can just use the website for the `source`.
```suggestion
            "source": "https://www.cricbuzz.com"
```